rotkehlchen/data/importer.py

```diff
@@ -934,3 +934,87 @@ def import_nexo_csv(self, filepath: Path) -> Tuple[bool, str]:
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_shapeshift_trade(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing only trades from ShapeShift.
+        """
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['timestamp'],
+            formatstr='iso8601',
+            location='ShapeShift',
+        )
+        buy_asset = symbol_to_asset_or_token(csv_row['outputCurrency'])
+        buy_amount = deserialize_asset_amount(csv_row['outputAmount'])
+        sold_asset = symbol_to_asset_or_token(csv_row['inputCurrency'])
+        sold_amount = deserialize_asset_amount(csv_row['inputAmount'])
+        rate = deserialize_asset_amount(csv_row['rate'])
+        fee = deserialize_fee(csv_row['minerFee'])
+        in_addr = csv_row['inputAddress']
+        out_addr = csv_row['outputAddress']
+        # TODO: determine exact time ShapeShift started using SAI with trades in csv exports
+        sai_sunset_timestamp = deserialize_timestamp_from_date(
+            date='2019-11-18T00:00:00Z',
+            formatstr='iso8601',
+            location='ShapeShift',
+        )
+        notes = f'''
```

Comment:
nitpick: Here and in some other places in the code I see you use `'''`. For the three quotes we actually use `"""` in the codebase.